app/build.gradle

```diff
@@ -54,11 +54,27 @@ android {
 }
 
 dependencies {
+    def composeBom = platform('androidx.compose:compose-bom:2022.10.00')
```

Comment:
https://android-developers.googleblog.com/2022/10/whats-new-in-jetpack-compose.html
https://developer.android.com/jetpack/compose/setup#groovy